### PR TITLE
Parser::Format::IPA: Shorten test names

### PR DESCRIPTION
**Pull request by cfconrad**
Remove all redundant info from test name and display the shortest possible.

A name like:
```
  EC2/test_sles_ec2_services.py::test_sles_ec2_services[paramiko://0.0.0.0-cloud-config.service]
```
becomes:
```
  EC2_test_sles_ec2_services_cloud-config
```

Ticket: https://progress.opensuse.org/issues/52208

### CODE REVIEW

File: lib/OpenQA/Parser/Format/IPA.pm [hunk 1]
```diff
@@ -37,14 +38,35 @@ sub parse {
...
+            if ($+{param}) {
+                my $param = $+{param};
+                $param =~ s/\.service$//;
+                $t_name .= '_' . $param;
+            }
+        }
+
+        # If a tests was triggered twice, we need to unique the name
+        if (exists($unique_names{$t_name})) {
+            $t_name .= '_' . sprintf("%02d", ++$unique_names{$t_name});
```
Comment (Martchus) [inline lib/OpenQA/Parser/Format/IPA.pm]: I'd move the '_' into the `sprintf`-string.

Comment (cfconrad) [reply lib/OpenQA/Parser/Format/IPA.pm]: thx, done

**Comment by okurz** [comment]
please correct the typos in the git commit message

**Comment by cfconrad** [comment]
I think it's ok now.

**Pull request merged**